File: Lista2/C/Lsala2/Ex9.c
#include <stdio.h> 

int main()
{
	int n1, n2;
	printf("1o Numero: ");
	scanf("%i", &n1);
	printf("2o Numero: ");
	scanf("%i", &n2);
	if(n1%4==0 || n1%5==0 || n2%4==0 || n2%5==0){
		if(n1%4==0 || n1%5==0){
			printf("O numero %i e divisivel por 4 ou por 5", n1);
		}
		if(n2%4==0 || n2%5==0){
			printf("O numero %i e divisivel por 4 ou por 5", n2);
		}
	}else{
		printf("Nenhum dos numeros e divisivel por 4 ou por 5");
	}
}

File: Lista2/Python/Lsala2/Ex9.py
n1 = int(input("1º Número: "))
n2 = int(input("2º Número: "))
if n1 % 4 == 0 or n1 % 5 == 0 or n2 % 4 == 0 or n2 % 5 == 0:
    if n1 % 2 == 0 or n1 % 3 == 0:
        print(f"O número {n1} é divesível por 4 ou por 5")
    if n2 % 2 == 0 or n2 % 3 == 0:
        print(f"O número {n2} é divesível por 4 ou por 5")
else:
    print("Nenhum dos números é divisível por 4 ou por 5")
